src/public/less/components/package-file-browser.less

```diff
@@ -70,16 +70,14 @@
 				}
 			}
 
-			.fa {
-				color: #000;
-				margin-right: 5px;
-				width: 13px;
-
-				&.fa-clipboard {
-					float: right;
-					margin-right: 20px;
-					margin-top: 3px;
-					cursor: pointer;
+			.actions-dropdown {
+				float: right;
+				margin-right: 20px;
+				margin-top: 3px;
+				cursor: pointer;
+
+				.fa {
```

Comment:
This needs to be outside `.actions-dropdown` because it also applied to file icons on the left.